datalad/support/tests/test_gitrepo.py

```diff
@@ -414,10 +414,10 @@ def test_GitRepo_fetch(test_path, orig_path, clone_path):
 
     fetched = clone.fetch(remote='origin')
     # test FetchInfo list returned by fetch
-    eq_([u'origin/master', u'origin/new_branch'],
+    eq_([u'origin/' + clone.get_active_branch(), u'origin/new_branch'],
         [commit.name for commit in fetched])
 
-    ok_clean_git(clone.path, annex=False)
+    ok_clean_git(clone.path)
```

Comment:
In other words: `ok_clean_git` was originally assuming, that the determination of whether or not a repo is clean does not depend on whether or not it is an annex. `annex=` added just a little additional test. But this fundamental assumption is not true.